App_Data/SwaggerSpecs/Microsoft.GuestConfiguration/guestconfiguration.json

```diff
@@ -1,1285 +1,1281 @@
           },
-          "x-ms-long-running-operation": true,
-          "produces": [
-            "application/json"
-          ],
-          "consumes": [
-            "application/json"
-          ]
-        },
-        "get": {
-          "tags": [
-            "GuestConfigurationAssignments"
-          ],
-          "operationId": "GuestConfigurationAssignments_Get",
-          "description": "Get information about a guest configuration assignment",
-          "x-ms-examples": {
-            "Get a guest configuration assignment": {
-              "$ref": "./examples/getGuestConfigurationAssignment.json"
-            }
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
           },
-          "parameters": [
-            {
-              "$ref": "../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
-            },
-            {
-              "name": "guestConfigurationAssignmentName",
-              "in": "path",
-              "required": true,
-              "type": "string",
-              "description": "The guest configuration assignment name."
-            },
-            {
-              "$ref": "../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
-            },
-            {
-                "$ref": "./definitions.json#/parameters/VmNameParameter"
-            },
-            {
-              "$ref": "../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
-            }
-          ],
-          "responses": {
-            "200": {
-              "description": "OK. The guest configuration assignment was returned successfully.",
-              "schema": {
-                "$ref": "#/definitions/GuestConfigurationAssignment"
-              }
-            },
-            "default": {
-              "description": "Error response describing why the operation failed.",
-              "schema": {
-                "$ref": "../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
-              }
-            }
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
           },
-          "produces": [
-            "application/json"
-          ],
-          "consumes": [
-            "application/json"
-          ]
-        },
-        "delete": {
-          "tags": [
-            "GuestConfigurationAssignments"
-          ],
-          "operationId": "GuestConfigurationAssignments_Delete",
-          "description": "Delete a guest configuration assignment",
-          "x-ms-examples": {
-            "Delete an guest configuration assignment": {
-              "$ref": "./examples/deleteGuestConfigurationAssignment.json"
-            }
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/VmNameParameter"
```

Comment:
if this type is not available in the common-types file, then we would need a local copy of this file in this folder and update the path to reference from that newly added file.